orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/model/NodeResult.java

```diff
@@ -0,0 +1,49 @@
+package com.sequenceiq.cloudbreak.orchestrator.model;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class NodeResult {
```

Comment:
name of this class might be to general, hard to guess the purpose of it, might be something `NodeAvailabilityResult` or `NodeReachabilityResult` or something similar

Reply:
You are right, I will fix this.